scripts/enable-ecs-agent-gpu-support.sh

```diff
@@ -130,6 +153,26 @@ if [[ $AMI_TYPE == "al2keplergpu" ]]; then
 
     sudo yum install -y cuda-toolkit-11-4
     echo "exclude=*nvidia* *cuda*" | sudo tee -a /etc/yum.conf
+elif [[ $AMI_TYPE == "al2023"*"gpu" ]]; then
+    kernel_release=$(uname -r)
+
+    sudo dnf install -y kernel-devel-"$kernel_release" \
+        kernel-modules-extra-"$kernel_release" \
+        nvidia-driver \
+        nvidia-fabric-manager \
+        pciutils \
+        xorg-x11-server-Xorg \
+        oci-add-hooks \
```

Comment:
this shouldn't be needed - the NVIDIA container toolkit will add the hooks, likewise for the next two `libnvidia-container*` deps, those should be handled by the container toolkit

Reply:
you can see what we do with the EKS AL2023 NVIDIA AMI here https://github.com/awslabs/amazon-eks-ami/pull/1924 

Note: we dual ship both the proprietary NVIDIA driver and the open GPU kernel module and load the correct one during instance provisioning based on the GPU card. Older cards will require the proprietary driver, newer will required the open GPU kernel module